lib/index.js

```diff
@@ -1,3 +1,3 @@
 module.exports = function (bundler) {
-  bundler.addAssetType('svg', require.resolve('./asset.js'));
+  bundler.addAssetType('isvg', require.resolve('./asset.js'));
```

Comment:
are you proposing to replace svg with isvg?

Reply:
yes, using svg chould conflict with other technic, when developer write svg path in scss , vue or something, this project has big difference between file loader , it does not produce a new file, so make it has a unique extension avoiding conflict with other file loader, also sound reasonable.

Reply:
unless the asset extension hook can have known the svg module  import from which kind of file or in which way, this is the simple solution.

Reply:
Yeah the `isvg` thing seems odd to me, but it is pretty intuitive, as long as it's documented properly... My only issue is that I would be pulling `.svg` files from another package, and I would not be able to change the file extension within that package without copying the SVGs out...

@bung87, can you give a code example that shows where this plugin would mess up usage of `.svg` in SCSS or Vue? I'm not sure I understand, and it might be something worth elaborating on in the documenting for the `.isvg` extension.

Reply:
@lazd it's described in https://github.com/albinotonnina/parcel-plugin-inlinesvg/issues/13
as it's a asset any other loader would transpile the svg module to a file path when you import it, you can simply create a project that contains scss , set a background to a svg file,then in js file import it, or in vue SCF write style refers to it , as the developer intend to refers a file, but this plugin is about generate inline code , that's not the right use case, also other plugin process would give a path as result.

I understand your use case , it make sense.
if remains the svg extension, this plugin should detect user import it in which file, it makes more complicated, since other process would interupt, or like webpack add some command in file path when user import it `xxx.svg!inline` or something.

Reply:
about your use case ,maybe can use   `.flowconfig`  `module.name_mapper` https://parceljs.org/module_resolution.html#flow-with-absolute-or-tilde-resolution